Birthday-2024-Project/Scripts/LevelSetup.gd

```diff
@@ -3,8 +3,9 @@ class_name LevelSetup
 
 @export var levelData : Array[PieceSetup] = []
 @export var jsonData : String
+@export var author : String
 @export var message : String
-@export var artFileName : String
+@export var levelPreview : CompressedTexture2D
```

Comment:
I understand it is setup for miniature levels in buttons after they're beaten, right?

Reply:
Correct. It's already hooked up to apply the image texture. But button states aren't setup since I figured it'd be a separate task to manage progression